src/dispatch/case/models.py

```
@@ -199,10 +200,22 @@ def participant_emails(self) -> list:
         return [participant.individual.email for participant in self.participants]
 
     @hybrid_property
-    def total_cost(self):
+    def total_cost_classic(self):
         total_cost = 0
         if self.case_costs:
             for cost in self.case_costs:
+                if cost.case_cost_type.model_type == CostModelType.new:
+                    continue
+                total_cost += cost.amount
+        return total_cost
+
+    @hybrid_property
+    def total_cost_new(self):
+        total_cost = 0
+        if self.case_costs:
+            for cost in self.case_costs:
+                if cost.case_cost_type.model_type == CostModelType.classic:
+                    continue
                 total_cost += cost.amount
         return total_cost
```

nit: could fail fast here to reduce nesting and improve readability